### PR TITLE
[FIX] web: list exports its resIds and  form can use them for pager

### PR DESCRIPTION
**Pull request by kebeclibre**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/web/static/tests/webclient/actions/window_action_tests.js [hunk 1]
```diff
@@ -132,6 +132,32 @@ QUnit.module("ActionManager", (hooks) => {
         ]);
     });
 
+    QUnit.test("correct pager when coming from list", async (assert) => {
+        assert.expect(4);
```
Comment (aab-odoo) [inline addons/web/static/tests/webclient/actions/window_action_tests.js:136]: not necessary

Comment (aab-odoo) [reply addons/web/static/tests/webclient/actions/window_action_tests.js:136]: simple question: are you sure this wasn't already tested?

**Review by aab-odoo — APPROVED** [review]

**Comment by kebeclibre** [comment]
robodoo r+

**Comment by kebeclibre** [comment]
robodoo r+ retry

**Comment by kebeclibre** [comment]
robodoo r+

**Pull request closed without merging**